### DISCUSSION TITLE
how to get bounding box of a gltf file

### DISCUSSION THREAD

**Discussion by Kshitij-Jadhav** [iTwin/itwinjs-core#5208] [Q&A] [answered]
I am reading gltf file into a graphics object using readGltfGraphics() method. any ideas how can i get the bounding box for the the GLTF model.
i have tried to get through the range method in graphics but when i try scale down the gltf file and the load it the range is inversely affected  by it in  scale.

**Comment by pmconne** [comment]
We can add that, hang on.

**Reply by pmconne** [reply]
#5209

**Reply by pmconne** [reply]
#5211

**Comment by pmconne** [comment, marked as answer]
This change has been merged. It will be included in iTwin.js 4.0 which is expected to be released within a few weeks.
